data/remnant/remnant 2 cognizance.txt

```diff
@@ -223,13 +223,13 @@ mission "Remnant: Cognizance 5"
 		conversation
 			branch fleetkilled
 				has "remnant: cognizance: killed korath fleet"
-			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant and waves you forward as soon as you approach. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of Remnant in this group is wearing a thick leather neck wrap that extends right up to their hairline. It has a scaly texture and appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`
+			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant and he waves you forward as soon as you approach. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather neck wrap that extends right up to their hairline. It has a scaly texture and appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`
```

Comment:
```suggestion
			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant, and he waves you forward as soon as you approach. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather neck wrap that extends right up to their hairline. It has a scaly texture and appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`
```

Reply:
Is this supposed to be an Oxford comma or is there some other reason to add it?

Reply:
"He waves you forward..." is a separate independent clause joined with the and conjunction, which requires the comma.